python/protobuf.c

```diff
@@ -90,6 +91,49 @@ PyObject *PyUpb_ObjCache_Get(const void *key) {
   }
 }
 
+// -----------------------------------------------------------------------------
+// Arena
+// -----------------------------------------------------------------------------
+
+typedef struct {
+  PyObject_HEAD upb_arena* arena;
```

Comment:
is there supposed to be a `;` after HEAD?

Reply:
No, unfortunately it's a macro that already specifies the semicolon. :(  This causes the bad clang-format, and I didn't find a good solution to make clang-format do something reasonable here. Fixed manually.